app/models/game.rb

```diff
@@ -53,13 +62,17 @@ def find_players
   end
```

Comment:
One concept we might be able to abstract into its own class is something called `PreDealSetup`anything having to do before the cards are dealt can be handled here like `find_order` `set_blinds`.  I don't know if you have a `card_shuffle` method but that could go there too.